library/src/main/kotlin/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -461,4 +515,47 @@ internal class TransactionPayloadFragment :
         }
         return result
     }
+
+    private fun MutableList<TransactionPayloadItem>.getCollapsableOrDefault(): List<TransactionPayloadItem> {
+        val default = this
+
+        return if (viewModel.isUsingCollapsableJson) {
+            try {
+                mapToJsonElements()
+            } catch (t: JsonSyntaxException) {
+                Toast.makeText(context, t.message, Toast.LENGTH_LONG).show()
+                Logger.error(t.message ?: "Error when formatting json")
```

Comment:
I'd rather display the `"Error when formatting json"` in the Toast and dump the stacktrace in logcat instead